firestore-translate-text/PREINSTALL.md

```diff
@@ -9,11 +9,19 @@ You specify the desired target languages using ISO-639-1 codes. You can find a l
 
 If the original non-translated field of the document is updated, then the translations will be automatically updated, as well.
 
-#### Multiple Field Translations
+#### Multiple collections
```

Comment:
Not sure this is right. You can translate multiple fields by passing an object to the input field. 

but you also want to inform the user they can create multiple extensions if they want to target different collections. 

So I'd keep this title `#### Multiple Field Translations` & create another section for different collections.